### DISCUSSION TITLE
Is there a mod to prevent apps on taskbar from moving when a new app is opened?

### DISCUSSION THREAD

**Discussion by slrgt** [ramensoftware/windhawk-mods#678] [Q&A]
It's really annoying to contstantly have my apps shift to the left when a app that isn't pinned to the taskbar is opened. Is there any mod that prevents my apps on my taskbar from moving?

**Comment by m417z** [comment]
What do you mean? Post screenshots that demonstrate the behavior you see and the behavior you'd like to have.

**Reply by slrgt** [reply]
Ah sorry about that. Here's a video showing the behavior.
https://github.com/ramensoftware/windhawk-mods/assets/114287507/80f20c37-7a5d-4f7d-a285-951ff49b0d70
I'd like it if apps just opened to the right of the pinned apps without moving them over

**Reply by m417z** [reply]
You can just configure your taskbar to align to the left:
https://zapier.com/blog/windows-11-taskbar-left/
